### ISSUE TITLE
Native Mobile application built in

### ISSUE THREAD

**Issue by echostorm** [flectrahq/discussion#62]
I would like to suggest adding the ability to not only allow users to create websites but to just as easily create Mobile application with the same integration capability available for the website builder, basically we would have another builder called mobile app builder just like we have the standard app builder and website builder. The mobile app should be capable of integrating with eCommerce and website-forms-appbuilder etc + add push notifications as a new marketing option besides email newsletters for this mobile application.

for more clarification:
example mobile hybrid app service would be app-presser for wordpress and woocomerce I think something similar would be a great addon for flectra although would prefer more native application approach since we are using python should checkout frameworks that use python for mobile app development...

**Comment by rm-jawengo** [comment]
I would prefer a solution for progressive web app (see https://developers.google.com/web/progressive-web-apps/) as this will be the next great step forward. I think this could be a good solution for using flectrahq with some basic offline features without having to develop special apps for different hardware.

There is a funny and perfect explanation on it at youtube: https://www.youtube.com/watch?v=cmGr0RszHc8&t=2125s

**Comment by parthivgls** [comment]
the only point of concern with progressive web app is this : https://m.phillydevshop.com/apples-refusal-to-support-progressive-web-apps-is-a-serious-detriment-to-future-of-the-web-e81b2be29676

**Comment by rm-jawengo** [comment]
Yes I have seen this but in a german post there is a link which shows that service workers will be implemented in webkit. You can see the state here: https://webkit.org/status/#specification-service-workers (in development)

null
